hooks/post-checkout

```diff
@@ -23,8 +24,17 @@ if [[ -n "${BUILDKITE_PLUGIN_TRIVY_SEVERITY:-}" ]] ; then
   export args+=("--severity ${BUILDKITE_PLUGIN_TRIVY_SEVERITY}")
   echo "using non-default severity types"
 fi
+
+if [[ -n "${BUILDKITE_PLUGIN_TRIVY_SECURITY_CHECKS:-}" ]] ; then
+	  export fsargs+=("--security-checks ${BUILDKITE_PLUGIN_TRIVY_SECURITY_CHECKS}")
+	    echo "using $BUILDKITE_PLUGIN_TRIVY_SECURITY_CHECKS security checks"
+	  else
+	   echo "using default security checks"
+		  export fsargs+=("--security-checks vuln,config")
+fi
```

Comment:
The code seems wrongly formatted, do we have a bash linter running? the `else` statement looks off

Reply:
I will change the PR title and let you know. The latest build is working yesterday also,lint is working fine .
Please find the below builtekite log reference.

https://buildkite.com/equinix/trivy-buildkite-plugin/builds/41#0182131c-6ca0-4795-b3d3-4c300471a8a4
![image](https://user-images.githubusercontent.com/99889761/179746132-c23f3db6-adb7-401c-9825-a043635d8d87.png)

Reply:
OK, can you make sure there is consistent whitespacing from lines 28 to 34? I'll research if there's a formatting linter we can use to avoid these issues.

Reply:
For time being I've added the 'shellcheck' and corrected the formatting ,Can you please check.
Also, update the PR title.